karapace/key_format.py

```diff
@@ -0,0 +1,57 @@
+"""
+karapace - Key correction
+Copyright (c) 2022 Aiven Ltd
+See LICENSE for details
+"""
+
+from enum import Enum
+from karapace.typing import JsonData
+from karapace.utils import json_encode
+from typing import Optional
+
+SCHEMA_KEY_ORDER = ["keytype", "subject", "version", "magic"]
+CONFIG_KEY_ORDER = ["keytype", "subject", "magic"]
+NOOP_KEY_ORDER = ["keytype", "magic"]
+
+CANONICAL_KEY_ORDERS = [SCHEMA_KEY_ORDER, CONFIG_KEY_ORDER, NOOP_KEY_ORDER]
+
+
+class KeyMode(Enum):
+    CANONICAL = 1
+    KARAPACE = 2
```

Comment:
It would be improvement to explain and rename this `KARAPACE` format, e.g. naming it as `OLD_KARAPACE` or `DEPRECATED_KARAPACE`.